web/src/views/HomeView/SingleStat.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <v-container
+    fluid
+    class="py-0 px-8"
+    data-id="stat"
```

Comment:
I'm not sure how you want to go with this, but even for dynamic things, you could use `class` ( https://vuejs.org/v2/guide/class-and-style.html ):
```
class="da-stat"
:class="`da-stat-${name}`"
```